### DISCUSSION TITLE
custom-domain setup

### DISCUSSION THREAD

**Discussion by mayank0202** [upptime/upptime#938] [General] [answered]
How to setup the cname?
like i have a qualified domain `mayankdevops.com`
so can i make a subdomain and add in uptimemetrics.yaml file?
subdomain like `uptime.mayankdevops.com`?

**Comment by AnandChowdhary** [comment, marked as answer]
https://upptime.js.org/docs/configuration#custom-domain

**Comment by mayank0202** [comment]
Yeaa i have done this 
Thanks @AnandChowdhary
